includes/currency-functions.php

```diff
@@ -513,16 +513,41 @@ function give_get_currency_name( $currency_code ) {
  *
  * @since 1.0
  *
- * @param string $price           The donation amount.
- * @param string $currency_code   The currency code.
- * @param bool   $decode_currency Whether to decode the currency HTML format or not.
+ * @param string $price The donation amount.
+ * @param array  $args  It accepts 'currency_code', 'decode_currency' and 'form_id'.
  *
  * @return mixed|string
  */
-function give_currency_filter( $price = '', $currency_code = '', $decode_currency = false ) {
+function give_currency_filter( $price = '', $args = array() ) {
 
-	if ( empty( $currency_code ) || ! array_key_exists( (string) $currency_code, give_get_currencies() ) ) {
-		$currency_code = give_get_currency();
+	// Bailout.
+	if ( empty( $price ) || ( ! is_numeric( $price ) && ! is_string( $price ) ) ) {
+		return $price;
+	}
+
+	// Get functions arguments.
+	$func_args = func_get_args();
+
+	if ( isset( $func_args[1] ) && ( is_string( $func_args[1] ) || is_numeric( $func_args[1] ) ) ) {
```

Comment:
@emgk `$func_args[1]` is currency_code why are you check this as `is_numeric`

Reply:
@ravinderk thanks, I have removed `is_numeric` checking on this.